evm_loader/cli/src/main.rs

```diff
@@ -163,7 +163,7 @@ async fn execute<'a>(
             let storage_account =
                 pubkey_of(params, "storage_account").expect("storage_account parse error");
             cancel_trx::execute(
-                context.rpc_client,
+                context.rpc_client.as_any().downcast_ref().unwrap(),
```

Comment:
Honestly I don't like this `as_any` usage, but as far as I can see, it is a common pattern in our code-base. Perhaps it still would be better to use `expect` here?

Reply:
Avoiding downcasting entirely is difficult because the Rpc type is not known at compile time on all code paths.

The code can be further redactored to remove some more downcasting, but the main purpose of the PR is to reduce the Rpc trait to the actual methods which are implemented on more than one type.